inc/taskjobstate.class.php

```diff
@@ -118,7 +118,13 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     {
         switch ($item->getType()) {
             case 'Computer':
-                return __("Tasks / Groups", "glpiinventory");
+                if (
+                    method_exists($item, 'getInventoryAgent')
```

Comment:
Method must exist on `Computer`, it seems useless to check it

Reply:
Yes, but without this check the CI is not happy:

https://github.com/glpi-project/glpi-inventory-plugin/pull/489/commits/e0567abd3e45d4a46a66de16d08aaf43e4e638c4

Reply:
it can be solved using `/** @var Computer $item` I guess

Reply:
(not a blocker anyway)